### PR TITLE
Add config option for jwt at

### PR DESCRIPTION
**Pull request by nsklikas**
IAM-886

Add config option for jwt access tokens

### CODE REVIEW

**Review by BarcoMasile — APPROVED** [review]
LGTM, just a super small suggestion on a description

File: config.yaml [hunk 1]
```diff
@@ -9,6 +9,11 @@ options:
       Available values are: panic, fatal, error, warn, info, debug, and trace.
     type: string
     default: "info"
+  jwt_access_tokens:
+    description:
+      Issue JWT access tokens, defaults to opaque access tokens
```
[the remark this answers is not in the document]
Comment (natalian98) [reply config.yaml:14]: `access_token_strategy=jwt|opaque` would be more intuitive, but I'll leave it up to you

[the remark this answers is not in the document]
Comment (nsklikas) [reply config.yaml:14]: I don't like having to write strings for binary config, also what happens if the user writes a typo

[the remark this answers is not in the document]
Comment (natalian98) [reply config.yaml:14]: That's fair, what about `jwt_access_tokens_enabled`? Current config name doesn't suggest it's a binary config

**Comment by nsklikas** [comment]
Changed the config to default to `True`

**Review by natalian98 — DISMISSED** [review]

**Review by BarcoMasile — DISMISSED** [review]

**Review by BarcoMasile — APPROVED** [review]

**Pull request merged**